README.md

```diff
@@ -76,24 +76,11 @@ For the github version make sure you are on the develop branch.
 1. Follow [these instructions](https://github.com/watson-developer-cloud/unity-sdk#configuring-your-service-credentials) to create your Speech To Text and Conversation services and find their credentials (using [IBM Cloud](https://console.ng.bluemix.net/)).
```

Comment:
This URL should be `https://console.bluemix.net/` so it's not region specific (current URL is north america).

Reply:
done